code/obj/item/gun/kinetic.dm

```diff
@@ -483,6 +528,77 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic/single_action)
 		..()
 		return
 
+/obj/item/gun/kinetic/survival_rifle
+	name = "\improper Efnysien survival rifle"
+	desc = "A semi-automatic rifle, renowned for it's easily convertible caliber, developed by Mabinogi Firearms Company."
+	icon = 'icons/obj/large/48x32.dmi'
+	icon_state = "survival_rifle_22"
+	item_state = "survival_rifle"
+	wear_state = "survival_rifle"
+	wear_image_icon = 'icons/mob/clothing/back.dmi'
+	force = MELEE_DMG_RIFLE
+	c_flags = NOT_EQUIPPED_WHEN_WORN | EQUIPPED_WHILE_HELD | ONBACK
```

Comment:
What are the first two flags for?

Reply:
Believe they're needed to properly allow worn on back behavior. They're used on every single gun that you can wear on your back:
https://github.com/goonstation/goonstation/blob/484bd57f3e09e2dbc22c126ea070f718fbf0107f/_std/defines/clothing.dm#L47-L50

Reply:
Pretty sure those flags are for guns that have slowdown when wielded, like the nukie LMG. Assuming this one doesnt have slowdown, you should be OK to remove them.

Reply:
Interesting, I'll remove them and test.

Reply:
> Pretty sure those flags are for guns that have slowdown when wielded, like the nukie LMG. Assuming this one doesnt have slowdown, you should be OK to remove them.

Just tested, seems to have no discernable difference, so I'll just remove them.